---
layout: home
# title: Home
# navigation_weight: 1
---

<style type="text/css">
.image-left {
  display: block;
  margin-left: 20px;
  margin-right: auto;
  float: right;
}
.spaced-lines {line-height: 20pt;} 
</style>

![right-aligned-image](headshot5.png){: .image-left width="315px"}
<div markdown="1" class="spaced-lines">
I am a postdoctoral fellow at the [Eric and Wendy Schmidt Center](https://www.ericandwendyschmidtcenter.org/) at the [Broad Institute](https://www.broadinstitute.org/).
I completed my PhD in Computer Science at [Princeton University](https://www.cs.princeton.edu), where I was very fortunate to be advised by [Ben Raphael](https://www.cs.princeton.edu/~braphael/) 
and generously supported by an [NSF Graduate Research Fellowship](https://www.cs.princeton.edu/news/computer-science-phd-students-awarded-nsf-graduate-research-fellowship) and a [Siebel Scholar award](https://www.cs.princeton.edu/news/graduate-students-receive-siebel-scholar-awards). 
    

I am broadly interested in developing statistical and machine learning methods to address problems in biology, with a particular interest in problems involving spatial or network structure.    
     
<!-- [[Google Scholar](https://scholar.google.com/citations?user=JPKTNnMAAAAJ&hl=en&oi=ao), [CV](cv_website.pdf), [Github](https://github.com/uthsavc)] -->

Here is [my CV](cv_website.pdf) and [Google Scholar](https://scholar.google.com/citations?user=JPKTNnMAAAAJ&hl=en&oi=ao).
</div>
<!-- &nbsp; -->

## Recent Updates

- (September 2024) Selected as a [Rising Star in Data Science](https://datascience.ucsd.edu/rising-stars-in-data-science/) and will attend the workshop at UC San Diego in November.

- (July 2024) New pre-print on [measuring higher-order epistasis](https://www.biorxiv.org/content/10.1101/2024.07.17.603976v1).

- (June 2024) [GASTON](https://www.biorxiv.org/content/10.1101/2023.10.10.561757v1), our deep learning model for learning spatial gradients from sparse spatial transcriptomics data, was accepted to _Nature Methods_.

- (May 2024) [DIALECT](https://www.biorxiv.org/content/10.1101/2024.04.24.590995v1), our method to identify mutually exclusive driver mutations that accounts for passenger mutations, was accepted to and won the **Best Paper Award** at *RECOMB-CCB 2024*.

- (March 2024) [Copulacci](https://academic.oup.com/bioinformatics/article/40/Supplement_1/i481/7700859), our method for learning cell-cell interactions from sparse spatial transcriptomics data, was accepted to *ISMB 2024*.

- (March 2024) I defended my PhD thesis! Slides available [here](presentations/FPO_actual.pdf). Next up: I will join the [Eric and Wendy Schmidt Center](https://www.ericandwendyschmidtcenter.org/) at the Broad Institute as a postdoctoral fellow.

<!-- - (December 2023) GASTON was accepted to RECOMB 2024, and I will present it in Boston in May.

- Posters/talks on Belayer and GASTON at: NCI Spring School (April 2023), NCI Junior Investigator Meeting (August 2023), Rutgers-Princeton cancer symposium (October 2023), CSHL Single-Cell Analyses meeting (November 2023).

- (November 2023) Just released our new algorithm GASTON, which uses deep learning to derive a topographic map of spatial gene expression. Check out the [pre-print](https://www.biorxiv.org/content/10.1101/2023.10.10.561757v1) and [code](https://github.com/raphael-group/GASTON)! -->

---    
<!-- &nbsp; -->

## Papers

<sup>\* denotes joint first authorship.</sup>     

[<span style="color:#c869bf">GASTON-Mix: a unified model of spatial gradients and domains using spatial mixture-of-experts.</span>](https://uthsavc.github.io/)     
**Uthsav Chitra**, Shu Dan, Fenna Krienen, Benjamin J. Raphael   
In submission.

[<span style="color:#c869bf">Mapping the topography of spatial gene expression with interpretable deep learning.</span>](https://www.biorxiv.org/content/10.1101/2023.10.10.561757v1)     
**Uthsav Chitra**, Brian J. Arnold, Hirak Sarkar, Cong Ma, Sereno Lopez-Darwin, Kohei Sanno, Benjamin J. Raphael   
*Nature Methods*, in press. Previously appeared at *RECOMB 2024* [[slides](presentations/gaston_recomb2024_pres.pdf)].

[<span style="color:#c869bf">Quantifying higher-order epistasis: beware the chimera.</span>](https://www.biorxiv.org/content/10.1101/2024.07.17.603976v1)     
**Uthsav Chitra\***, Brian J. Arnold\*, Benjamin J. Raphael   
*Nature Communications*, in press.

[<span style="color:#c869bf">A count-based model for delineating cell-cell interactions in spatial transcriptomics data.</span>](https://academic.oup.com/bioinformatics/article/40/Supplement_1/i481/7700859)     
Hirak Sarkar\*, **Uthsav Chitra\***, Julian Gold, Benjamin J. Raphael   
*Bioinformatics* (2024). Accepted to *ISMB 2024*.

[<span style="color:#c869bf">A latent variable model for evaluating mutual exclusivity between driver mutations in cancer.</span>](https://uthsavc.github.io/)     
Ahmed Shuaibi\*, **Uthsav Chitra\***, Benjamin J. Raphael   
RECOMB Satellite Workshop on Computational Cancer Biology (*RECOMB-CCB 2024*).    
<ins>***Best Paper Award***.</ins>

[<span style="color:#c869bf">Belayer: Modeling discrete and continuous spatial variation in gene expression from spatially resolved transcriptomics.</span>](https://www.biorxiv.org/content/10.1101/2022.02.05.479261v1)     
Cong Ma\*, **Uthsav Chitra\***, Shirley Zhang, Benjamin J. Raphael   
*Cell Systems* (2022). Accepted to *RECOMB 2022*.

[<span style="color:#c869bf">NetMix2: Unifying network propagation and altered subnetworks.</span>](https://www.biorxiv.org/content/10.1101/2022.01.31.478575v1)     
**Uthsav Chitra\***, Tae Yoon Park\*, Benjamin J. Raphael     
*Journal of Computational Biology* (2022). Accepted to *RECOMB 2022* [[slides](presentations/netmix2_recomb2022_pres.pdf)].

[<span style="color:#c869bf">Quantifying and Reducing Bias in Maximum Likelihood Estimation of Structured Anomalies</span>](https://arxiv.org/abs/2007.07878)     
**Uthsav Chitra**, Kimberly Ding, Jasper C. H. Lee, Benjamin J. Raphael  
International Conference on Machine Learning (*ICML 2021*) [[slides](presentations/ICML_2021_pres.pdf), [ICML talk](https://slideslive.com/38958812/quantifying-and-reducing-bias-in-maximum-likelihood-estimation-of-structured-anomalies?ref=speaker-17429-latest)].    

[<span style="color:#c869bf">NetMix: A network-structured mixture model for reducing bias in the identification of altered subnetworks.</span>](https://www.biorxiv.org/content/10.1101/2020.01.18.911438v1)    
Matthew A. Reyna\*, **Uthsav Chitra\***, Rebecca Elyanow, Benjamin J. Raphael   
Journal of Computational Biology (2021). Accepted to *RECOMB 2020* [[slides](presentations/recomb2020_pres.pdf)].    

[<span style="color:#c869bf">Analyzing the Impact of Filter Bubbles on Social Network Polarization.</span>](https://arxiv.org/abs/1906.08772)    
**Uthsav Chitra**, Christopher Musco    
ACM Conference on Web Search and Data Mining (*WSDM 2020*).    
Preliminary version presented at KDD WISDOM 2019 workshop [[slides](presentations/kdd_presentation.pdf)].     

[<span style="color:#c869bf">Random Walks on Hypergraphs with Edge-Dependent Vertex Weights.</span>](https://arxiv.org/abs/1905.08287)          
**Uthsav Chitra**, Benjamin J. Raphael  
International Conference on Machine Learning (*ICML 2019*) [[slides](https://icml.cc/media/Slides/icml/2019/101(13-11-00)-13-12-15-5196-random_walks_on.pdf), [ICML talk @ 55:27](https://slideslive.com/38917909/optimization-and-graphical-models)].

[<span style="color:#c869bf">Committee Selection is More Similar Than You Think: Evidence from Avalanche and Stellar.</span>](https://arxiv.org/abs/1904.09839)     
Tarun Chitra, **Uthsav Chitra**  
Manuscript, 2019.

&nbsp;

---    
&nbsp;

## Personal

I like to go [<span style="color:#c869bf">bouldering</span>](http://instagram.com/uthsav_climbs/), attempt to do crosswords, and make bad puns.