---
layout: home
# title: Home
# navigation_weight: 1
---

<style type="text/css">
.image-left {
  display: block;
  margin-left: 20px;
  margin-right: auto;
  float: right;
}
.spaced-lines {line-height: 20pt;} 
</style>

<div class="image-icon-wrapper">
    <img src="images/headshot5.png" alt="headshot" class="image-left" width="250px" style="border: 0px solid black;">

    <div class="icon-container">
        <a href="https://scholar.google.com/citations?user=JPKTNnMAAAAJ&hl=en&oi=ao" target="_blank">
            <i class="ai ai-google-scholar ai-2x"></i>
        </a>
        <a href="cv_website.pdf" target="_blank">
            <i class="ai ai-cv ai-2x"></i>
        </a>
        <a href="https://github.com/uthsavc" target="_blank">
            <i class="fab fa-github-square fa-2x"></i>
        </a>
        <a href="https://x.com/uthsavc" target="_blank">
            <i class="fab fa-twitter-square fa-2x"></i>
        </a>
        <a href="https://orcid.org/0000-0001-6016-0960" target="_blank">
            <i class="ai ai-orcid ai-2x"></i>
        </a>
        <a href="mailto:uchitra@broadinstitute.org" target="_blank">
            <i class="fa-solid fa-envelope fa-2x"></i>
        </a>
    </div>
</div>

<div markdown="1" class="spaced-lines">


I am a postdoctoral fellow at the [Eric and Wendy Schmidt Center](https://www.ericandwendyschmidtcenter.org/) at the [Broad Institute of MIT and Harvard](https://www.broadinstitute.org/).


I completed my PhD in Computer Science at [Princeton University](https://www.cs.princeton.edu). I was very fortunate to be advised by [Ben Raphael](https://www.cs.princeton.edu/~braphael/) 
and supported by an [NSF Graduate Research Fellowship](https://www.cs.princeton.edu/news/computer-science-phd-students-awarded-nsf-graduate-research-fellowship) and a [Siebel Scholar award](https://www.cs.princeton.edu/news/graduate-students-receive-siebel-scholar-awards). 
    

I am broadly interested in developing statistical and machine learning methods to address fundamental problems in biology, with a particular focus on problems involving graphs or spatial structure.    
    

Here is [my CV](cv_website.pdf), [Google Scholar](https://scholar.google.com/citations?user=JPKTNnMAAAAJ&hl=en&oi=ao), and my [research statement](CS_statement_5_no_future_work.pdf) (minus the "future work" section). 


**In Fall 2025, I will be starting as an Assistant Professor in the [Department of Computer Science](https://www.cs.jhu.edu/) at [Johns Hopkins University](https://www.jhu.edu/). Please [contact me](mailto:uthsav.chitra@gmail.com) if you are interested in working with me!**

</div>
<!-- &nbsp; -->

## Recent Updates

- (March 2025) Two papers (GASTON-Mix, and SARDINE by Gary Hu) accepted to ISMB 2025.

- (March 2025) GLACIER, a method for spatial Granger causality (with Prannav Shankar, Huan Liang, Rohit Singh) was accepted to _RECOMB-seq 2025_. Pre-print coming soon!

- (February 2025) Our paper on [measuring higher-order epistasis](https://www.biorxiv.org/content/10.1101/2024.07.17.603976v1) was published in _Nature Communications_.

- (January 2025) [GASTON](https://www.nature.com/articles/s41592-024-02503-3), an interpretable deep learning model for learning spatial gradients from sparse spatial gene expression data, was published in _Nature Methods_.

- (November 2024) Selected as a [Rising Star in Data Science](https://datascience.ucsd.edu/rising-stars-in-data-science/) and attended the workshop at UC San Diego.

- (May 2024) [DIALECT](https://www.biorxiv.org/content/10.1101/2024.04.24.590995v1), our method to identify mutually exclusive driver mutations that accounts for passenger mutations, was accepted to and won the **Best Paper Award** at *RECOMB-CCB 2024*.

<!-- - (March 2024) [Copulacci](https://academic.oup.com/bioinformatics/article/40/Supplement_1/i481/7700859), our method for learning cell-cell interactions from sparse spatial transcriptomics data, was accepted to *ISMB 2024*. -->



## Papers

<sup>\* denotes joint first authorship     
† denotes joint last (senior) authorship</sup>

[<span style="color:#c869bf">Spatial metabolic gradients in the liver and small intestine.</span>](https://uthsavc.github.io/)    
Laith Samarah, Clover Zheng, ..., **Uthsav Chitra**, ...,  Benjamin J. Raphael, Joshua Rabinowitz.    
In review.

[<span style="color:#c869bf">Decoding the causal drivers of spatial cellular topology.</span>](https://uthsavc.github.io/)     
Prannav Shankar, Huan Liang, **Uthsav Chitra†**, Rohit Singh†.    
RECOMB Satellite Conference on Biological Sequence Analysis (*RECOMB-seq 2025*).

[<span style="color:#c869bf">Anomaly Detection in Spatial Transcriptomics via Spatially Localized Density Comparison.</span>](https://uthsavc.github.io/)     
Gary Hu, Julian Gold, **Uthsav Chitra**, Sunay Joshi, Benjamin J. Raphael.    
Accepted to *ISMB 2025*.

[<span style="color:#c869bf">GASTON-Mix: a unified model of spatial gradients and domains using spatial mixture-of-experts.</span>](https://www.biorxiv.org/content/10.1101/2025.01.31.635955v1)     
**Uthsav Chitra**, Shu Dan, Fenna Krienen, Benjamin J. Raphael   
Accepted to *ISMB 2025*.    

[<span style="color:#c869bf">A latent variable model for evaluating mutual exclusivity between driver mutations in cancer.</span>](https://www.biorxiv.org/content/10.1101/2024.04.24.590995v1)     
Ahmed Shuaibi\*, **Uthsav Chitra\***, Benjamin J. Raphael       
In preparation.    
<ins>***Best Paper Award***</ins> at RECOMB Satellite Workshop on Computational Cancer Biology (*RECOMB-CCB 2024*).  

[<span style="color:#c869bf">Mapping the topography of spatial gene expression with interpretable deep learning.</span>](https://www.nature.com/articles/s41592-024-02503-3)     
**Uthsav Chitra**, Brian J. Arnold, Hirak Sarkar, Kohei Sanno, Cong Ma, Sereno Lopez-Darwin, Benjamin J. Raphael   
*Nature Methods* (2025). Accepted to *RECOMB 2024* [[RECOMB slides](presentations/gaston_recomb2024_pres.pdf), [MIA lightning talk](https://youtu.be/y747azUJOnI?t=2060)].

[<span style="color:#c869bf">Resolving discrepancies between chimeric and multiplicative measures of higher-order epistasis.</span>](https://www.nature.com/articles/s41467-025-56986-5)     
**Uthsav Chitra\***, Brian J. Arnold\*, Benjamin J. Raphael   
*Nature Communications* (2025).    

[<span style="color:#c869bf">A count-based model for delineating cell-cell interactions in spatial transcriptomics data.</span>](https://academic.oup.com/bioinformatics/article/40/Supplement_1/i481/7700859)     
Hirak Sarkar\*, **Uthsav Chitra\***, Julian Gold, Benjamin J. Raphael   
*Bioinformatics* (2024). Accepted to *ISMB 2024*.

[<span style="color:#c869bf">Belayer: Modeling discrete and continuous spatial variation in gene expression from spatially resolved transcriptomics.</span>](https://www.biorxiv.org/content/10.1101/2022.02.05.479261v1)     
Cong Ma\*, **Uthsav Chitra\***, Shirley Zhang, Benjamin J. Raphael   
*Cell Systems* (2022). Accepted to *RECOMB 2022*.

[<span style="color:#c869bf">NetMix2: Unifying network propagation and altered subnetworks.</span>](https://www.biorxiv.org/content/10.1101/2022.01.31.478575v1)     
**Uthsav Chitra\***, Tae Yoon Park\*, Benjamin J. Raphael     
*Journal of Computational Biology* (2022). Accepted to *RECOMB 2022* [[RECOMB slides](presentations/netmix2_recomb2022_pres.pdf)].

[<span style="color:#c869bf">Quantifying and Reducing Bias in Maximum Likelihood Estimation of Structured Anomalies</span>](https://arxiv.org/abs/2007.07878)     
**Uthsav Chitra**, Kimberly Ding, Jasper C. H. Lee, Benjamin J. Raphael  
International Conference on Machine Learning (*ICML 2021*) [[ICML slides](presentations/ICML_2021_pres.pdf), [ICML talk](https://slideslive.com/38958812/quantifying-and-reducing-bias-in-maximum-likelihood-estimation-of-structured-anomalies?ref=speaker-17429-latest)].    

[<span style="color:#c869bf">NetMix: A network-structured mixture model for reducing bias in the identification of altered subnetworks.</span>](https://www.biorxiv.org/content/10.1101/2020.01.18.911438v1)    
Matthew A. Reyna\*, **Uthsav Chitra\***, Rebecca Elyanow, Benjamin J. Raphael   
*Journal of Computational Biology* (2021). Accepted to *RECOMB 2020* [[RECOMB slides](presentations/recomb2020_pres.pdf)].    

[<span style="color:#c869bf">Analyzing the Impact of Filter Bubbles on Social Network Polarization.</span>](https://arxiv.org/abs/1906.08772)    
**Uthsav Chitra**, Christopher Musco    
ACM Conference on Web Search and Data Mining (*WSDM 2020*).    
Preliminary version presented at *KDD WISDOM 2019* workshop [[KDD slides](presentations/kdd_presentation.pdf)].     

[<span style="color:#c869bf">Random Walks on Hypergraphs with Edge-Dependent Vertex Weights.</span>](https://arxiv.org/abs/1905.08287)          
**Uthsav Chitra**, Benjamin J. Raphael  
International Conference on Machine Learning (*ICML 2019*) [[ICML slides](https://icml.cc/media/Slides/icml/2019/101(13-11-00)-13-12-15-5196-random_walks_on.pdf), [ICML talk @ 55:27](https://slideslive.com/38917909/optimization-and-graphical-models)]. 

[<span style="color:#c869bf">Committee Selection is More Similar Than You Think: Evidence from Avalanche and Stellar.</span>](https://arxiv.org/abs/1904.09839)     
Tarun Chitra, **Uthsav Chitra**    
Manuscript, 2019.    


&nbsp;

---    
&nbsp;

## Personal

I like to go [<span style="color:#c869bf">bouldering</span>](http://instagram.com/uthsav_climbs/) in my free time.